src/pages/gd/Claim/index.tsx

```diff
@@ -222,7 +219,7 @@ your G$. 🙂`,
                             firstName="Test"
                             method="redirect"
                             claim={handleClaim}
-                            claimed={claimed}
+                            claimed={{ claimed, state }}
```

Comment:
1. should be a separate prop
2. I do not like we're passing a whole txState. Let's add a prop `claiming` and will pass `state?.status === "Mining"` as its value